polkadot-protocol/architecture/system-chains/asset-hub.md

```diff
@@ -1,4 +1,203 @@
+
+Integrating with Asset Hub offers several key benefits, particularly for infrastructure providers and users:
+
+- **Support for non-native on-chain assets** - Asset Hub enables seamless asset creation and management, allowing projects to develop tokens or assets that can interact with the broader ecosystem
+- **Lower transaction fees** - Asset Hub offers significantly lower transaction costs—approximately one-tenth of the fees on the relay chain, providing cost-efficiency for regular operations
+- **Reduced deposit requirements** - depositing assets in Asset Hub is more accessible, with deposit requirements that are around one one-hundredth of those on the relay chain
+- **Payment of transaction fees with non-native assets** - users can pay transaction fees in assets other than the native token (DOT or KSM), offering more flexibility for developers and users
+
+Assets created on the Asset Hub are stored as part of a map, where each asset has a unique ID that links to information about the asset, including details like:
+
+- The management team
+- The total supply
+- The number of accounts holding the asset
+- Sufficiency for account existence - whether the asset alone is enough to maintain an account without a native token balance
+- The metadata of the asset, including its name, symbol, and the number of decimals for representation
+
+Some assets can be regarded as sufficient to maintain an account's existence, meaning that users can create accounts on the network without needing a native token balance (i.e., no existential deposit required). Developers can also set minimum balances for their assets. If an account's balance drops below the minimum, the balance is considered dust and may be cleared.
+
+## Assets Pallet
+
+The Polkadot SDK's Assets pallet is a powerful module designated for creating and managing fungible asset classes with a fixed supply. It offers a secure and flexible way to issue, transfer, freeze, and destroy assets. The pallet supports various operations and includes permissioned and non-permissioned functions to cater to simple and advanced use cases.
+
+Visit the [Assets Pallet Rust docs](https://paritytech.github.io/polkadot-sdk/master/pallet_assets/index.html){target=\_blank} for more in-depth information.
+
+### Key Features
+
+Key features of the Assets pallet include:
+
+- **Asset issuance** - allows the creation of a new asset, where the total supply is assigned to the creator's account
+- **Asset transfer** - enables transferring assets between accounts while maintaining a balance in both accounts
+- **Asset freezing** - prevents transfers of a specific asset from one account, locking it from further transactions
+- **Asset destruction** - allows accounts to burn or destroy their holdings, removing those assets from circulation
+- **Approval-based transfers** - a non-custodial mechanism to enable one account to approve a transfer of assets on behalf of another
```

Comment:
Do you want to give this the same name you give it down below?
**Non-Custodial Transfers (Approval API)**